package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chs-dev",
-  "version": "1.3.0",
+  "version": "2.0.0",
   "author": "Damian Dunajski (ddunajski@companieshouse.gov.uk)",
```

Comment:
by the way, apart from the historical value, the original author has not much meanig now

Reply:
Addressed in #105 - good spot 👍🏼